pamelalozano/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,39 @@
+
+package com.google.sps.servlets;
+
+
+import java.util.*;  
+import com.google.gson.Gson;
+import java.io.IOException;
+
+public class Comment {
+  
+  private String subject;
+  private String msg;
+  private Date date;
+
+  public Comment(String s, String m, Date d) {
+    this.subject = s;
+    this.msg = m;
+    this.date = d;
+  }
+  
+  public void setSubject(String s) {
+      this.subject = s;
+  }
+  public void setMessage(String m) {
+      this.msg = m;
+  }
+  public void setDate(Date d) {
+      this.date = d;
+  }
+  public String getSubject() {
+      return this.subject;
+  }
+  public String getMessage() {
+      return this.msg;
+  }
+  public Date getDate() {
+      return this.date;
+  }
```

Comment:
optional. A common pattern is for these kinds "data" classes to have methods that return different serializations of themselves e.g. to its JSON representation, to a String, etc. For example:
```
public String toJson() {
  return new Gson().toJson(this);
}
```

When you have an instance of `Comment`, you can just call `toJson()` to get its JSON representation instead of passing it to Gson#toJson.
```
Comment comment = new Comment("test", "test comment", new Date());
String commentJson = comment.toJson();
```

Reply:
Cool will do! Thanks :)